src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -132,11 +132,17 @@ function getListBlockType(
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
+  // Some blocks must be treated as unstyled when not present on the blockRenderMap
   const unstyledElement = blockRenderMap.get('unstyled').element;
-  return blockRenderMap
+  const defaultUnstyledSet = new Immutable.Set(['p']);
```

Comment:
Would this be better resolved by allowing the configuration to specify the set of elements that should map *to* a given block type, not just block type to element? I.e. `unstyled: { element: ..., wrapper: ..., inputTags: ['div', 'p']}`